go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/confluentinc/cli/v3
 
-go 1.21
+go 1.21.4
```

Comment:
You don't need to include the patch version here. This is updated automatically by service bot
```suggestion
go 1.21
```

Reply:
Oh, sorry. My newest rebased messed up this. Fixed now